PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/console/SparkScalaLivyConsoleRunConfiguration.kt

```diff
@@ -60,7 +62,8 @@ class SparkScalaLivyConsoleRunConfiguration(project: Project,
     override fun getState(executor: Executor, env: ExecutionEnvironment): RunProfileState? {
         val session = SparkSession(
                 name,
-                URI.create(JobUtils.getLivyBaseUri(cluster).toString()),
+                URI.create(cluster.connectionUrl)
+                        .resolve(if (cluster.isEmulator || cluster is HDInsightLivyLinkClusterDetail) "/" else "/livy/"),
```

Comment:
I suppose the `HDInsightLivyLinkClusterDetail.getConnectionUrl()` should retrun the string ending with `/livy/`?